### ISSUE TITLE
[archive] daily: CV/DL research briefing 2026-09-17

### ISSUE THREAD

**Issue by Jung-woojin** [Jung-woojin/cv-research-radar#4]
# CV/DL 데일리 브리핑 — 2026-09-17

최근 공개·업데이트된 CV/VLM/DL 연구 중 실제 후속 연구 가능성과 재현성을 기준으로 선별했다.

## 오늘의 전체 흐름

1. RGB→event transfer에서 단순 token alignment보다 **semantic relation preservation**이 중요해지고 있다.
2. Remote sensing grounding은 단순 segmentation을 넘어 **visual drift + textual drift**를 함께 다루는 cross-domain benchmark로 확장 중이다.
3. Long-video VLM은 full-frame 입력보다 **macro-to-micro adaptive token allocation**이 핵심 효율화 방향으로 올라왔다.
4. VLM hallucination 완화는 answer accuracy보다 **semantic-spatial agreement / evidence grounding**을 직접 다루는 쪽으로 이동 중이다.
5. Representation 연구에서는 “더 좋은 feature를 만들자”보다 **정보가 이미 존재하는데 output이 못 읽는가?**라는 진단 관점이 강해지고 있다.

---

### Hyper-RED: Scalable Event Pre-training via Semantic Hypergraph Distillation
**v1: 2026-09-15**

DINOv3 RGB teacher의 semantics를 event encoder로 옮기되, token-wise 동일화를 강제하지 않고 **semantic hypergraph relation**을 distill한다. modality-specific structure를 보존하면서 cross-modal transfer하는 것이 핵심이다.

연구 연결점:
- RGB ↔ event
- camera degradation ↔ clean visual encoder
- multimodal distillation에서 feature equality보다 relation preservation

당시 공식 repo는 있었지만 README가 사실상 **“Code is coming.”** 상태라 즉시 재현 가능한 artifact는 아니었다.

---

### VPRef: A Cross-Domain Benchmark for Referring Remote Sensing Image Segmentation
**v1: 2026-09-15**

Vaihingen/Potsdam 기반 **46,972 image-language-mask triplet**을 만들고 language complexity를 Basic / Moderate / Complex로 나눴다.

핵심은 domain shift를:
- visual drift
- textual logic drift

로 분리한 점이다.

SAM3+LoRA baseline은 전체 parameter의 약 1.08%만 수정한다.

사용자 연구와의 연결:
- aerial/UAV
- grounding
- remote sensing cross-domain generalization
- foundation model PEFT

---

### LePoKet
효율적인 robotic vision을 위한 lightweight perception 방향. 최신 foundation model의 대형화를 그대로 따라가기보다 **embedded/robotics deployment target**을 우선한다는 흐름의 사례.

---

### PanoGS-SLAM
Gaussian Splatting 계열 SLAM이 point/mesh 기반 map을 대체하는 흐름의 연장선. 중요 포인트는 photorealistic rendering 자체보다 mapping representation이 downstream perception/relocation과 결합되는 방향이다.

---

### Noise2Noise Revisited
핵심 finding은 loss function보다 **training pair distribution**이 훨씬 더 큰 inductive bias를 만들 수 있다는 것.

L1/L2 차이는 제한적이지만 synthetic pair에서 real paired data로 넘어갈 때 gain이 훨씬 크게 나타난다.

연구적으로는:
> architecture/loss를 바꾸기 전에 data-generating process를 진단하라

는 메시지가 강하다.

---

### Video-HolmesV2: Can MLLMs Reason with Spatio-Temporal Audio-Visual Evidence in Long Videos?
**v1: 2026-09-15 · ECCV 2026**

10–150분 long video에서 단순 answer뿐 아니라:
- temporal evidence
- spatial evidence
- audio evidence

를 함께 요구한다.

강한 proprietary model도 60% 미만에 머물러, long-video QA가 단순 language prior로 해결되지 않음을 보였다.

핵심 변화:
```
answer accuracy
→ evidence-grounded reasoning
```

---

### OPD-Aha: From Linguistic Momentum to Visual Reflection in Multimodal On-Policy Distillation

잘못된 linguistic prefix가 누적되면서 teacher까지 같은 방향으로 끌리는 문제를, **real-image teacher vs visual-null teacher** 차이로 분리해 visual corrective signal을 만든다.

핵심 질문:
> VLM이 틀린 이유가 reasoning 부족인가, visual evidence를 실제로 쓰지 않았기 때문인가?

---

### VideoMM: Adaptive Macro-Micro Inference for Efficient Video MLLMs

저해상도 **Macro Proxy**로 전체 video를 먼저 보고, 필요한 region/time만 원본 **Micro Token**으로 복원한다.

LongVideoBench에서 full-context baseline 대비:
- 약 6.13× speedup
- +7.4% accuracy

를 보고.

중요한 점은 token pruning이 아니라 **coarse semantic filtering → selective high-resolution reasoning** 구조다.

---

### Semantic-Spatial Agreement Hallucination Mitigation

VLM hallucination을 language confidence만으로 잡지 않고, generated semantics와 actual spatial evidence의 agreement를 높이는 방향.

이는 향후 grounded VQA와 결합 가치가 높다.

---

### OPEN-1B
대규모 open training recipe/weights 흐름. 단순 parameter 공개보다 **data + training recipe + reproducibility**가 점점 평가 대상이 되고 있다.

### LoopSpec
Speculative / iterative inference efficiency 계열. 모델 크기 축소보다 **inference path 자체를 바꾸는 시스템 연구**가 커지고 있다.

### Temporal Credit Distillation
long-horizon sequence에서 어느 시점의 observation/action이 결과에 영향을 줬는지 credit assignment를 개선하는 방향.

---

# 오늘의 중요한 Finding 3개

## 1. Cross-modal transfer는 “같아지게 만들기”보다 “관계를 보존하기”로 이동
Hyper-RED가 대표적. modality 고유 정보를 죽이지 않으면서 semantic structure를 공유하는 것이 중요하다.

## 2. VLM evaluation은 answer에서 evidence로 이동
Video-HolmesV2와 hallucination 관련 연구를 보면, 앞으로 VQA는 “맞혔는가”보다 “어떤 frame/region 때문에 맞혔는가”가 중요해질 가능성이 높다.

## 3. 효율화는 uniform compression이 아니라 adaptive allocation
VideoMM처럼 전체 context를 동일 해상도로 처리하지 않고, coarse proxy를 이용해 필요한 부분만 비싼 computation을 쓰는 방식이 강해지고 있다.

---

# 오늘 바로 읽을 Top 3

1. **VPRef** — aerial/grounding/cross-domain이라는 사용자의 관심사와 가장 직접적.
2. **Video-HolmesV2** — 향후 Video VLM benchmark 설계 관점에서 중요.
3. **Hyper-RED** — representation transfer를 architecture보다 relation 관점에서 보는 좋은 예.

---

# 후속 연구 씨앗

### 1. Cross-domain grounding에서 visual drift와 language drift를 독립적으로 측정
Remote sensing에만 한정하지 않고 maritime CCTV에서:
- 같은 visual scene, 다른 질문 complexity
- 같은 질문, 다른 port/weather domain

을 분리해 generalization matrix를 만들 수 있다.

### 2. Video VQA evidence benchmark
Answer + timestamp + ROI + object ID를 동시에 요구해 shortcut을 줄인다.

### 3. Adaptive high-resolution perception
전체 frame을 고해상도로 처리하지 않고 coarse scout가 필요한 region만 선택하되, tiny-object recall이 무너지지 않도록 risk constraint를 넣는 방향.

---

# 오늘 안 읽어도 되는 것

새 backbone 자체보다 기존 foundation representation을 어떻게 **distill / route / ground / selectively compute**할지가 오늘 더 중요한 흐름이었다.
